chart/templates/clusterrole.yaml

```diff
@@ -39,7 +39,11 @@ rules:
     resources: ["nodes", "nodes/status"]
     verbs: ["update", "patch"]
   {{- end }}
-  {{- if or .Values.controlPlane.advanced.virtualScheduler.enabled .Values.sync.toHost.pods.hybridScheduling.enabled }}
+  {{- if or
+      (and .Values.controlPlane.distro.k8s.enabled .Values.controlPlane.distro.k8s.scheduler.enabled)
```

Comment:
.Values.controlPlane.distro.k8s.enabled is false by default but implicitly enabled if no other distro is enabled